src/behavior.rs

```diff
@@ -3,13 +3,18 @@ use bevy::prelude::*;
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
 
+/// how the touch stick is positioned onscreen
+///
+/// - `Fixed`: Static position onscreen
+/// - `Floating`: spawn at pressed point
+/// - `Dynamic`: follow point on drag
```

Comment:
```suggestion
```

This just duplicates what's below

Reply:
i duplicated it because of how rust-analyzer shows struct docs
but honestly it is not _that_ useful